src/content/docs/es/guides/deploy/google-firebase.mdx

```diff
@@ -4,33 +4,127 @@ description: Cómo desplegar tu proyecto de Astro usando Firebase Hosting de Goo
 type: deploy
 i18nReady: true
 ---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 [Firebase Hosting](https://firebase.google.com/products/hosting) es un servicio provisto por la plataforma de desarrollo de Google [Firebase](https://firebase.google.com/), el cual puede ser usado para desplegar un proyecto de Astro.
 
 Consulta nuestra guía para [desplegar en el servicio de Hosteo de Firebase](/es/guides/backend/google-firebase/).
 
+## Configuración de proyecto
+
+Tu proyecto de Astro puede ser desplegado en Firebase como un sitio estático o como un sitio renderizado del lado del servidor (SSR).
+
+### Sitio estático
+
+Tu proyecto de Astro es un sitio estático por defecto. No necesitas ninguna configuración extra para desplegar un sitio estático de Astro en Firebase.
+
+### Adaptador para SSR
+
+To enable SSR in your Astro project and deploy on Firebase add the [Node.js adapter](/es/guides/integrations-guide/node/). 
+
+:::note
+Deploying an SSR Astro site to Firebase requires the [Blaze plan](https://firebase.google.com/pricing) or higher.
+Desplegar un sitio SSR de Astro en Firebase requiere el [plan Blaze](https://firebase.google.com/pricing) o superior.
+:::
+
 ## Cómo desplegar
 
-1. Asegúrate de tener [firebase-tools](https://www.npmjs.com/package/firebase-tools) instalado.
+1. Instala la [CLI de Firebase](https://github.com/firebase/firebase-tools). Esta es una herramienta de línea de comandos que te permite interactuar con Firebase desde la terminal.
+
+  <PackageManagerTabs>
+    <Fragment slot="npm">
+    ```shell
+    npm install firebase-tools
+    ```
+    </Fragment>
+    <Fragment slot="pnpm">
+    ```shell
+    pnpm install firebase-tools
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn add firebase-tools
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+2. Autentica la CLI de Firebase con tu cuenta de Google. Esto abrirá una ventana de navegador donde puedes iniciar sesión con tu cuenta de Google.
+
+  <PackageManagerTabs>
+    <Fragment slot="npm">
+    ```shell
+    npx firebase login
+    ```
+    </Fragment>
+    <Fragment slot="pnpm">
+    ```shell
+    pnpm exec firebase login
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn firebase login
+    ```
+    </Fragment>
+  </PackageManagerTabs>
```

Comment:
```suggestion
    <PackageManagerTabs>
      <Fragment slot="npm">
      ```shell
      npx firebase login
      ```
      </Fragment>
      <Fragment slot="pnpm">
      ```shell
      pnpm exec firebase login
      ```
      </Fragment>
      <Fragment slot="yarn">
      ```shell
      yarn firebase login
      ```
      </Fragment>
    </PackageManagerTabs>
```

Reply:
I won't do the rest, but I can if you want me to!